src/Generator.cpp

```diff
@@ -21,362 +26,655 @@
 #include "HGCalSimulation/FastShower/interface/Event.h"
 #endif
 
+using namespace std;
+
+Generator::Generator(const Parameters& params): geometry_(params.geometry()),
+                                                output_(params.general().output_file),
+                                                shower_(params.shower()),
+                                                parameters_(params) { 
+}
+
+
+Generator::~Generator(){
+}
+
+
+double** Generator::readCalibration(const string filename) {
+
+    double** calib = new double*[52];
```

Comment:
There is a `new`; but who is responsible to delete this.
Except if really needed, don't use `new`. Here use an `std::array` instead.

Reply:
Fixed